addons/pos_restaurant/static/src/overrides/models/models.js

```diff
@@ -42,6 +42,13 @@ patch(Order.prototype, {
     setCustomerCount(count) {
         this.customerCount = Math.max(count, 0);
     },
+    amountPerGuest() {
```

Comment:
```suggestion
    amountPerGuest(numCustomers = this.customerCount) {
```
this way the code in the numberPopup feedback will be much simpler